src/index.js

```diff
@@ -17,7 +17,7 @@ const basemap = L.tileLayer(config.basemap_url, {
     errorTileUrl: config.basemap_error_url,
     noWrap: true,
     minNativeZoom: config.minZoom,
-    maxNativeZoom: config.maxZoom,
+    maxNativeZoom: config.maxZoom-1,
```

Comment:
config.maxZoom - 1 is not really maintainable. It's better to keep all of this in the config file. Could you create config.minNativeZoom and config.maxNativeZoom in config.js in the source image properties section (lines 1-16) and config.minZoom and config.maxZoom in the map properties section (lines 18-28)?